src/citrine/_serialization/properties.py

```diff
@@ -22,22 +22,50 @@
 
 
 class Property(typing.Generic[DeserializedType, SerializedType]):
+    """
+    Basic operating unit of a serialization layer.
+
+    By defining a Property as a class variable, access methods are defined and
+    the
+
+    Parameters
+    ----------
+    serialization_path: str
+        The location in the serialized structure where the value of interest can be
+        found.  The value can be nested in the structure, where separate keys are
+        joined by `.`.  Required in a top level property, but not necessary if
+        a property is wrapped by a container.
+    serializable: bool
+        Include the field when serializing an object to a dictionary; default: True
+    deserializable: bool
+        Look for the field when deserializing a dictionary into an object; default: True
+    override: bool
+        Use a parent class' accessor methods; default: False
+    use_init: bool
+        When deserializing, use the __init__ method instead of the setter
+
+    """
 
     def __init__(self,
                  serialization_path: typing.Optional[str] = None,
+                 *,
                  serializable: bool = True,
                  deserializable: bool = True,
                  default: typing.Optional[DeserializedType] = None,
-                 override: bool = False
+                 override: bool = False,
+                 use_init: bool = False
```

Comment:
New parameter.  When de-serializing an object, use klass(args) instead of __new__.  Necessary for cases where, for example, a private field is set to None in the __init__ routine.